spec/amp-var-substitutions.md

```diff
@@ -350,6 +350,12 @@ For instance:
 
 ## Miscellaneous
 
+### AMP_VERSION
+
+Provides a string with the AMP release version. The prefix denotes canary(`00`), production(`01`) or an experiment version ( &gt; `01`).
```

Comment:
I'd rather just have the version without prefix. It's easier to compare 2 versions and know which one is newer. When looking up the version number without prefix it can be found in the git tags as is. I wasn't sure if anyone else would need the prefix though. 
Split the canary vs prod prefix into another variable?

@avimehta, which style would be most useful for you?